hack/cluster/linux.json

```diff
@@ -0,0 +1,52 @@
+{
+    "apiVersion": "vlabs",
+    "location": "",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "1.14",
+            "kubernetesConfig": {
+                "useCloudControllerManager": true,
+                "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.14.0",
```

Comment:
Should replace with `mcr.microsoft.com/k8s/core/azure-cloud-controller-manager:v0.1.0`